README.md

```diff
@@ -8,9 +8,18 @@ or Chaos Communication Congress.
 
 The app is available for Android devices.
 
+Bug reports, feature requests can be submitted as an [issue](https://github.com/EventFahrplan/EventFahrplan/issues).
+Please read the instructions in the [contribution guide](https://github.com/EventFahrplan/EventFahrplan/blob/master/CONTRIBUTING.md) in order to contribute to this project.
```

Comment:
Simplification: `https://github.com/EventFahrplan/EventFahrplan/blob/master/CONTRIBUTING.md` can be a relative path as in `CONTRIBUTING.md`.

Reply:
I added the three changes in this commit
03f97a3